network-store-client/src/main/java/com/powsybl/network/store/client/PreloadingNetworkStoreClient.java

```diff
@@ -57,6 +57,12 @@ public class PreloadingNetworkStoreClient extends AbstractForwardingNetworkStore
     private final NetworkCollectionIndex<Set<ResourceType>> cachedResourceTypes
             = new NetworkCollectionIndex<>(() -> EnumSet.noneOf(ResourceType.class));
 
+    private final NetworkCollectionIndex<Map<ResourceType, Set<String>>> cachedExtensionAttributesByTypeAndName
```

Comment:
rename to cachedExtensionNamesByResourceType

Reply:
Simplified this in https://github.com/powsybl/powsybl-network-store/pull/411/commits/1d2c57e75f0193c6d358372294460f2872ac1d4d 
It was already handled in CollectionCache.